visualization/dashboard/src/MLIDashboard/Controls/DependencePlot/DependencePlot.styles.ts

```diff
@@ -73,7 +73,7 @@ export const dependencePlotStyles: () => IProcessedStyleSet<IDependencePlotStyle
         },
         secondaryChartPlacolderBox: {
             height: "400px",
-            width: "100%"
+            flex: 1
```

Comment:
interesting!  haven't seen this before.  are there other flexible items?  Is this referring to: https://www.w3schools.com/cssref/css3_pr_flex.asp
how does this impact the chart as opposed to width=100%?

Reply:
it is referring to flex! here is a helpful guide https://css-tricks.com/snippets/css/a-guide-to-flexbox/
@imatiach-msft